.github/dependabot.yml

```diff
@@ -3,4 +3,6 @@ updates:
   - package-ecosystem: github-actions
     directory: "/"
     schedule:
-      interval: weekly
\ No newline at end of file
+      interval: weekly
+    ignore:
+      - dependency-type: "all"  # Ensures only GitHub Actions are updated
```

Comment:
This `ignore` is still under `package-ecosystem: github-actions`, so it would likely not prevent further npm dependency bumps.

You should probably use this instead

```
  - package-ecosystem: "npm"
    ignore:
      - dependency-type: "all"  # Ensures only GitHub Actions are updated
```

But you might want to reconsider updating npm automatically via dependabot - doing rolling upgrades like this might make it easier to maintain this extension, compared to doing major upgrades at a more infrequent pace. You can limit how many PRs can be open at a time, so that you don't get flooded with a lot of PRs and all the updates are rolled into 1-2 PRs.

Reply:
I though about this when I saw dependanbot updating the version of the packages @rdimaio. However,  even simple package upgrades break the installation of the package (I haven't go into much details, to be honest, tho). So the upgrade of the extension would require a bit more of attention.

We also have a hard requirement (`notebook<7`), otherwise we would need to touch many more underlying infrastructure components.